CONTRIBUTING.md

```diff
@@ -3,6 +3,37 @@
 We'd love your help! Please join our weekly [SIG
 meeting](https://github.com/open-telemetry/community#special-interest-groups).
 
+## Target audiences
+
+The OpenTelemetry Collector has two main target audiences:
+
+1. End-users, aiming to use an OpenTelemetry Collector binary as a middleware on their infra, sitting between their
+workloads and the telemetry data storage.
+1. Downstream distributions, which can be an official one as part of the OpenTelemetry Collector project, such as the
```

Comment:
I know markdown does the right thing, but please use 2 :))

Reply:
"Downstream" is a bit confusing term, if possible let's avoid using it.
```suggestion
1. Collector distributions, which depend on OpenTelemetry Collector "core" repository. Distributions can be an official OpenTelemetry community project, such as the
```

Reply:
Great minds think alike, see https://github.com/open-telemetry/opentelemetry-collector/pull/4723#discussion_r789882513

Reply:
I replaced "Downstream distributions" with "Collector distributions", as I don't think there's a missing context with this change and it's likely clearer to a broader audience. Thanks for the suggestion.